packs/boons-and-curses/shizuru-major-curse.json

```diff
@@ -20,7 +20,18 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Gods & Magic"
         },
-        "rules": [],
+        "rules": [
+            {
+                "definition": [
```

Comment:
Is this comprehensive enough to be worthwhile?

Reply:
Yeah, we need to rethink how we do these breath weapon things but we'll cross that bridge later.